recipes/cvxopt/build.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+
+if [ "$(uname)" == "Linux" ];
+then
+    # SuiteSparse needs the Real Time library on Linux.
+    export CFLAGS="${CFLAGS} -lrt"
```

Comment:
Was failing the examples on Linux otherwise. So, we had to add this flag. This library doesn't exist on Mac and SuiteSparse doesn't try to link to it so we are fine there.